homeassistant/components/solaredge/models.py

```diff
@@ -2,20 +2,12 @@
 from __future__ import annotations
 
 from dataclasses import dataclass
-from datetime import datetime
 
+from homeassistant.components.sensor import SensorEntityDescription
```

Comment:
For the future, I'd try to keep the sensor integration imports in the sensor platform.

Reply:
Why? We do these things all the time, e.g., in constants files.

Reply:
Constant files are only for constants that need to be shared between more than one module in the integration. Platform specific names are better to keep within each platform. It's nice to keep the import branching as clean as possible.

Reply:
I don't agree on that. I'd like to move as much out of the platforms as possible.

And honestly, we do this all the time, for example, look at all the coordinator files.

Putting this in the constant file is ugly too as it is not a constant, but it is defined there most integrations. Leaving it in sensor, means sensor is important const, and const is importing sensor; which is circular.

Reply:
The coordinator module here has clean import branches. It's only the const.py and models.py that have unclean imports since they import from the sensor integration.

Reply:
Sorry. I'm afraid I'm not able to get on the same page with this.